source/javascripts/meetups.js

```diff
@@ -45,7 +45,9 @@
 
   handler.buildMap(mapOptions, function() {
     if(navigator.geolocation) {
-      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap);
+      var geoLocation = navigator.geolocation.getCurrentPosition(drawMap, function(){
```

Comment:
In that case, can't you just do:

```js
navigator.geolocation.getCurrentPosition(drawMap, drawMap)
```

Why does the second need to be an anonymous function that just calls a function with no args?